.github/ISSUE_TEMPLATE/tc2_bug_report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Team Comtress bug report
+about: A bug that occurs in Team Comtress
```

Comment:
maybe specify that this is for bugs that occur **only** in TC2 here, since this is seen before the user clicks to see the rest of the template

Reply:
I actually deliberately didn't do that. We can fixup the labels later during triage, and I don't want to have a giant paragraph explaining that TF2 reports are both TF2 and TC2 in the description.

Basically the indented behavior is people choose whichever of the two templates and we take it from there.